artists.php

```diff
@@ -0,0 +1,297 @@
+<?php
```

Comment:
Isn't this story 2?

Reply:
No? Story 2 is to get all songs by an artist. The artists.php file?
artists.php get's all the artists and their albums?